csv_export_invoice/tests/test_export.py

```diff
@@ -100,3 +100,17 @@ def test_invoice_csv_export_exports_unicode_data(self):
 
         # needs mocking
         # ice.action_send_to_backend_base()
+
+    def test_invoice_csv_export_non_exporter(self):
+        # coverage
+        for invoice in self.env["account.invoice"].search([]):
+            invoice.export_to_sftp = datetime.datetime.now()
```

Comment:
I think adding a comment here to explain why you set all the `export_to_sftp` would be useful.

Reply:
comment added